apps/ui/src/networks/starknet/actions.ts

```diff
@@ -230,6 +245,8 @@ export function createActions(
 
       if (relayerType && ['evm', 'evm-tx'].includes(relayerType)) {
         await verifyNetwork(web3, l1ChainId);
+      } else {
+        await verifyStarknetNetwork(web3, chainId);
```

Comment:
Looks like we only verify network for vote and propose (on Starknet we call `verifyNetwork` there because it's the only actions that Ethereum wallet can perform on Starknet networks), but on native network (Ethereum wallet on Ethereum network) every action is verified (see where we call `verifyNetwork` in `evm/actions`) - we should do the same here -> every action in starknet/actions.js should use it.